### PR TITLE
Add Node.js code samples to Stats and Tracing

### LINKED ISSUE #466: Add Node.js code snippets to Tracing/Metrics

### PR DESCRIPTION
**Pull request by hvent90**
Fixes #466

### CODE REVIEW

File: content/stats/measure.md [hunk 1]
```diff
@@ -41,8 +41,8 @@ Float64|Records float-like values
 
 ### Source code example
 
-{{<tabs Go Java Python CplusPlus>}}
```
Comment (odeke-em) [inline content/stats/measure.md]: Please keep these as they were, no need to radically change a style that has been used throughout, otherwise we have noisy and large PRs

File: content/stats/measure.md [hunk 2]
```diff
@@ -41,8 +41,8 @@ Float64|Records float-like values
...
-{{<highlight go>}}
```
Comment (odeke-em) [inline content/stats/measure.md]: Please keep this as was, no need for this change.

File: content/stats/measure.md [hunk 3]
```diff
@@ -110,9 +110,9 @@ def main():
 
 if __name__ == '__main__':
     main()
-{{</highlight>}}
+```
 
-{{<highlight cpp>}}
```
Comment (odeke-em) [inline content/stats/measure.md]: ditto

File: content/stats/measure.md [hunk 4]
```diff
@@ -139,7 +139,31 @@ int main(int argc, char** argv) {
             {{LinesMeasure(), 238}},
             {{BytesInMeasure(), 7000}});
 }
-{{</highlight>}}
+```
+
+```js
```
Comment (odeke-em) [inline content/stats/measure.md]: Please stick to the old style to make for a less noisy PR

File: content/stats/measure.md [hunk 5]
```diff
@@ -139,7 +139,31 @@ int main(int argc, char** argv) {
...
+import { Stats, MeasureUnit } from "@opencensus/core";
+
+// Our Stats manager
+const stats = new Stats();
+
+const mLatencyMs = stats.createMeasureDouble("latency", MeasureUnit.MS, "The latency in milliseconds");
+const mLinesIn = stats.createMeasureInt64("size", MeasureUnit.UNIT, "The number of lines processed");
```
Comment (odeke-em) [inline content/stats/measure.md]: This measure is not needed because in other samples we are merely applying a count aggregation to mLatencyMs which is recorded on every line. Please remove `mLinesIn`

File: content/stats/view.md [hunk 6]
```diff
@@ -57,7 +57,7 @@ func enableViews() error {
 
 	lineCountView = &view.View{
 		Name:        "myapp/lines_in",
-		Measure:     mLatencyMs,
```
Comment (odeke-em) [inline content/stats/view.md]: Please revert this change, we are applying a count aggregation on mLatencyMs which is invoked once for every line unless am mistaken. If mLatencyMs is used more than once, then we can just use mBytes*

File: content/stats/view.md [hunk 7]
```diff
@@ -102,7 +102,7 @@ public class JavaSnippet {
 
             View.create(Name.create("myapp/lines_in"),
                         "The number of lines that were received",
-                        M_LATENCY_MS,
```
Comment (odeke-em) [inline content/stats/view.md]: Please revert this change, `M_LATENCY_MS` with a count aggregation provides metrics for number of lines read in, assuming `M_LATENCY_MS` records latency once, if not, please use `M_BYTES_IN`.

File: content/stats/view.md [hunk 8]
```diff
@@ -134,17 +134,17 @@ def enable_views():
     line_count_view = view.View("myapp/lines_in",
                              "The number of liens that were received",
                              [key_method],
-                             m_latency_ms,
```
Comment (odeke-em) [inline content/stats/view.md]: Ditto, unnecessary

**Comment by hvent90** [comment]
@odeke-em done, and done! Thank you for taking a thorough look at the PR!

File: content/tracing/span/time_events/message_event.md [hunk 9]
```diff
@@ -96,6 +96,9 @@ serverEvent = time_event.MessageEvent(seqNumber, type=time_event.Type.SENT,
                 uncompressed_size_bytes=1024, compressed_size_bytes=512)
 span.add_time_event(time_event.TimeEvent(datetime.datetime.utcnow(), serverEvent))
 {{</highlight>}}
+
+{{<highlight js>}}
```
Comment (odeke-em) [inline content/tracing/span/time_events/message_event.md]: I guess this means message events are unimplemented?

Comment (hvent90) [reply content/tracing/span/time_events/message_event.md]: Thanks for commenting on this. At first glance looking through the node repo I didn't see anything about message events, but I just looked again and found a bunch of info. I'll push up another commit in a few minutes.

Comment (hvent90) [reply content/tracing/span/time_events/message_event.md]: Actually, upon looking at it again, I don't see anywhere to instantiate a message event. I only see [references](https://github.com/census-instrumentation/opencensus-node/blob/59bd2161ceeb60ddb8861f38675f42409088021e/packages/opencensus-core/src/trace/model/span-base.ts#L178-L184).

Comment (hvent90) [reply content/tracing/span/time_events/message_event.md]: Ill remove the js snippet for now
